contrib/database/sql/conn.go

```diff
@@ -135,6 +137,11 @@ func (tc *TracedConn) ExecContext(ctx context.Context, query string, args []driv
 		cquery, spanID := tc.injectComments(ctx, query, tc.cfg.dbmPropagationMode)
 		ctx, end := startTraceTask(ctx, QueryTypeExec)
 		defer end()
+		if appsec.RASPEnabled() {
```

Comment:
Discussed offline with @eliottness: ok to keep for now but I feel like this is pretty much useless and that we could just have RASP disabled by letting the dyngo events go through without appsec event listeners. Also, RASP is enabled by default and disabling it via env vars will be very unlikely.

So overall, I feel like those appsec.RASPEnabled() could be replaced by our good old appsec.Enabled() friend without requiring a RASP-specific one.

Reply:
dc217c2a6a893